man/adoc/bpftrace.adoc

```diff
@@ -4221,6 +4257,17 @@ The possible options are:
 - `warn` - enable feature but print a warning
 - `enable` - enable feature
 
+==== unstable_testing
```

Comment:
Curious to hear what @viktormalik and @amscanne think but I don't think we need to gate this feature. It's so benign and doesn't really pertain to actually debugging/investigations with bpftrace. If in the future we wanted to trash it, we could just issue a warning and not run them.

Reply:
Removed the `unstable_testing` config in the latest push.